site/content/guides/routing.md

```diff
@@ -0,0 +1,285 @@
+component or an app when no other route matched. To enable this behaviour add a `Route` without a 
+`path` to your group of `Route` components and them wrap them into `Switch`.
+
+```js
+...
+import { Route, Link, Switch } from 'frint-router-react';
+...
+const NotFoundPage = () => <div>Not found</div>;
+...
+const RootComponent = () => {
+  return (
+    <div>
+      ...
+      <div className="content">
+        <Switch>
+          <Route path="/" component={HomePage} exact />
+          <Route path="/about" component={AboutPage} />
+          <Route component={NotFoundPage} />
+        </Switch>
+      </div>
+    </div>
+  );
+};
+
+```  
+
+Now whenever you navigate to URL that doesn't match `/` or `/about` you'll see `NotFoundPage`
+rendered.
+
+Note: in `Switch` only the first matching `Route` gets rendered.
+
+## Nested routes
+
+It is also possible for child components to define their own subroutes. So for example if you want
+`AboutPage` to have subpages you can render `Link` and `Route` to it.
+
+```js
+...
+const AboutUs = () => <article>Some content about us...</article>;
+const AboutThem = () => <article>Some content about them...</article>;
+
+const AboutPage = ({ match }) => {
+  return (
+    <div>
+      <h2>About page</h2>
+      <ul>
+        <li><Link to={`${match.url}`}>About</Link></li>
+        <li><Link to={`${match.url}/us`}>About us</Link></li>
+        <li><Link to={`${match.url}/them`}>About them</Link></li>
+      </ul>
+      <Switch>
+        <Route path={`${match.url}/us`} component={AboutUs} />
+        <Route path={`${match.url}/them`} component={AboutThem} />
+      </Switch>
+    </div>
+  );
+};
+...
+```
+
+Because `AboutPage` is rendered using `Route` component it receives a `match` object as a prop. This
+object contains `url` which let's you define routes relative to the current.  
+
+It is important that the parent `Route` doesn't have `exact` prop so that it would match all the URLs
+starting with the `path`.
+
+## Route params
+
+In your path you can also define params which will be passed to your component once the path is 
+matched. With params previous example can be rewritten in the following way: 
+
+```js
+const AboutAnyone = ({ match }) => <article>Some content about {match.params.party}...</article>;
+
+const AboutPage = ({ match }) => {
+  return (
+    <div>
+      <h2>About page</h2>
+      <ul>
+        <li><Link to={`${match.url}`}>About</Link></li>
+        <li><Link to={`${match.url}/us`}>About us</Link></li>
+        <li><Link to={`${match.url}/them`}>About them</Link></li>
+      </ul>
+      <Switch>
+        <Route path={`${match.url}/:party`} component={AboutAnyone} />
+      </Switch>
+    </div>
+  );
+};
+```
+
+## Another example
+
+For more information about usage of Frint routing you can take a look at example here: 
+https://github.com/Travix-International/frint/tree/master/examples/router 
+
+To launch it clone `frint` repository
```

Comment:
I can understand about the reluctance towards installing something globally. I think the new `npx` can be handy in this case.

But since I see you are already pointing the user to the example directory in the repo, the example's README then should tell them enough. just like kitchensink's: https://github.com/Travix-International/frint/tree/router/examples/kitchensink

We should add a README for the `router` example there too.

And in the guides here, we can keep the `frint-cli` commands. I see more tools coming around `frint-cli` in near future. It would be good to promote it wherever we can early.